docs/getting-started-with-scalardb.md

```diff
@@ -1,24 +1,173 @@
 # Getting Started with ScalarDB
 
-Here we assume Oracle JDK 8 and the underlying storage/database such as Cassandra are properly configured.
-If you haven't done it, please configure them first by following [this](getting-started.md).
+This getting started tutorial explains how to configure your preferred database in ScalarDB and set up a basic electronic money application.
+
+{% capture notice--warning %}
+**Warning**
+
+This electronic money application is simplified for this tutorial and isn't suitable for a production environment.
+{% endcapture %}
+
+<div class="notice--warning">{{ notice--warning | markdownify }}</div>
+
+## Install a JDK
+
+Because ScalarDB is written in Java, you must have one of the following Java Development Kits (JDKs) installed in your environment.
+
+* [Oracle JDK 8](https://www.oracle.com/java/technologies/downloads/#java8)
+* [Oracle JDK 11](https://www.oracle.com/java/technologies/downloads/#java11)
+* [OpenJDK 8](https://openjdk.org/install/)
```

Comment:
Are there any other versions of the JDK that we should add here?

Reply:
Since there are many JRE/JDK distributions, I think it's hard to list them all comprehensively. So I think we can list some of them as representative examples.